### ISSUE TITLE
Creating an empty group using an existing app ID should return 409

### ISSUE THREAD

**Issue by pierlo-upitup** [mesosphere/marathon#3385]
When sending a `POST` to `/v2/groups` with a group `{"id": "/group/existing-appname"}` I’d expect to receive a `409`, but I am getting a `200` instead 

```
{
  deploymentId: "7a8ad237-a76b-428e-8915-5dc9c98716b3"
  version: "2016-03-02T11:02:45.586Z"
}
```

**Comment by alexanderweber** [comment]
@pierlo-upitup my reproduction steps are:
- POST on /v2/groups: `{ "id": "/group/existing-appname" }`. I do receive 201.
- POST on /v2/groups again: `{ "id": "/group/existing-appname" }`. I do receive 409.

Meaning everything is OK. Could you help me out with reproduction?

**Comment by pierlo-upitup** [comment]
start by creating an app with `/group/existing-appname` – ;-)

**Comment by alexanderweber** [comment]
@pierlo-upitup I did miss the _an existing app_ part of the title. Thanks for pointing it out.

I did fix this issue in #3398. A POST on /v2/group: `{ "id": "/group/existing-appname" }` now yields 409: `{
  "message": "An app with the path /group/existing-appname already exists."
}`

**Comment by pierlo-upitup** [comment]
Awesome! Thanks!
